cmd/datamon/cmd/root.go

```diff
@@ -49,6 +51,9 @@ var logFatalln = log.Fatalln
 var logFatalf = log.Fatalf
 var osExit = os.Exit
 
+// used to patch over calls to Authable.Principal() during test
+var authorizer auth.Authable = gauth.New()
```

Comment:
since `New()` is arbitrary procedure, not patching of static declaration, i suggest declaration here and assignment in `init()`, `initConfig()`, or one of the other existing lifecycle stages such as those in `flags.go`